pkg/plugins/models.go

```diff
@@ -197,3 +197,8 @@ func NewEnabledPlugins() EnabledPlugins {
 		Apps:        make([]*AppPlugin, 0),
 	}
 }
+
+type PluginErrorInfo struct {
+	ErrorCode string `json:"errorCode"`
+	PluginID  string `json:"pluginId,omitempty"`
+}
```

Comment:
@wbrowne can you have a look at these go files? Has there been any changes in your PR when you merged? I don't think these changes should be here as I haven't touched any backend code.

Reply:
Yeah exactly. These changes shouldn't be here. All my stuff is now merged to master - so you should just need to be merge/rebase and see all my changes

Reply:
eh, I guess it's a squash merge issue as this is now synced with master and the changes are still here